src/x509.c

```diff
@@ -8635,8 +8635,19 @@ WOLF_STACK_OF(WOLFSSL_X509)* wolfSSL_X509_chain_up_ref(
                 return WOLFSSL_FAILURE;
             }
 
+            cert->decodedCert = (DecodedCert*)XMALLOC(sizeof(DecodedCert), cert->heap,
```

Comment:
Instead of allocating a huge DecodedCert structure, wouldn't it make better sense to add a field to `struct Cert` -> `extBasicConstSet`? Avoid using decodedCert.